circle.yml

```diff
@@ -10,4 +10,5 @@ test:
     override:
         - ! gofmt -s -d . 2>&1 | read
         - gometalinter --cyclo-over=12 ./...
-        - go test -v -race -timeout=120s ./...
+        - go test -v -timeout=2m ./...
```

Comment:
The problem is that race enabled property based tests with the `testing/quick` package take FOREVER. Hence, those are skipped with the `-short` flag below.